### PR TITLE
Import in the readme

### PR DESCRIPTION
**Pull request by BenFradet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Currently available only for scala 2.11. In your build.sbt file,
 ```sbt
 resolvers += Resolver.jcenterRepo
 
-libraryDependencies += "io.ticofab" % "aws-request-signer_2.11" % "0.4.1"
+libraryDependencies += "io.ticofab" %% "aws-request-signer" % "0.4.1"
```
Comment (ticofab) [inline README.md:17]: if we do this, and someone uses this library from a scala 2.12 project, the library won't be found.

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ val awsCredentialProvider = new StaticCredentialsProvider(new BasicAWSCredential
 val service = "es"
 val region = "eu-central-1"
 def clock(): LocalDateTime = LocalDateTime.now(ZoneId.of("UTC"))
-val signer = AwsSigner(awsCredentialProvider, region, service, clock)
+val signer = io.ticofab.AwsSigner(awsCredentialProvider, region, service, clock)
```
Comment (ticofab) [inline README.md:32]: Why would you write this in the readme?

Comment (BenFradet) [reply README.md:32]: I personally had to go the source to check the package name

File: README.md (hunk 1 above)
Comment (BenFradet) [reply README.md:17]: how would you be able to use a library in a 2.12 project compiled against 2.11?

Comment (ticofab) [reply README.md:17]: You most likely wouldn't be able to. That is exactly why I want to make this visible in the README. I opened one issue about moving to 2.12 :)

File: README.md (hunk 2 above)
Comment (ticofab) [reply README.md:32]: Ok, this can be useful information then.

File: README.md (hunk 1 above)
Comment (BenFradet) [reply README.md:17]: I thought

> Currently available only for scala 2.11. In your build.sbt file,

would be clear enough, but I'm happy to remove it.

**Comment by ticofab** [comment]
I released 0.5.0 with these updates - including the package description. Thanks!

**Pull request closed without merging**